graphql.go

```diff
@@ -252,3 +252,21 @@ func getOperation(document *query.Document, operationName string) (*query.Operat
 	}
 	return op, nil
 }
+
+// GetQueryNames is tokopedia specific method to get all query names from query string
+func GetQueryNames(queryString string) []string {
```

Comment:
Pls add tests for this. I tried adding but getting some import issues in other part of the test file. Hopefully you will be able to resolve it.